docs/docs/forms.mdx

```diff
@@ -297,55 +297,51 @@ actions:
 - validate_restaurant_form
 ```
 
-When the form is executed it will run your custom action. In your custom action
-you can either
+When the form is executed it will run your custom action.
 
-- validate already extracted slots. You can retrieve them from the tracker by running
-  `tracker.form_slots_to_validate`.
```

Comment:
can we at least mention this in a note or at the end to let them know that they can use this to fully customize the validation? We are already seeing people who are trying to deactivate / activate forms for their validation methods and I think they'd be disappointed if they don't see a way to this with a regular `Action`

Reply:
I mentioned that there is `tracker.form_slots_to_validate` and it can be used to customize the validation process in the end of the section.